egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -636,10 +722,10 @@ def adjust_am_priors(dir, input_model, avg_posterior_vector, output_model,
                     avg_posterior_vector=avg_posterior_vector,
                     output_model=output_model))
 
-
 def compute_average_posterior(dir, iter, egs_dir, num_archives,
                               prior_subset_size, left_context, right_context,
-                              run_opts, get_raw_nnet_from_am=True):
+                              run_opts, get_raw_nnet_from_am=True,
+                              use_multitask_egs=False):
```

Comment:
OK, if you are not using this function then you should probably revert most of the changes to this function (and to calls of it), there is no point adding lines of code that couldn't work.  But isn't it problematic to rename just one output-name to "output" and do the average-posterior stuff?  Because you might care about >1 of the outputs....
I still don't understand exactly how you are doing the average-posterior computation in this multilingual setup.  If you could point me to some log files where you have run it, it might help.